src/smart-components/workspaces/WorkspaceDetail.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchWorkspace, fetchWorkspaces } from '../../redux/actions/workspaces-actions';
+import { ContentHeader } from '@patternfly/react-component-groups';
+import { Breadcrumb, BreadcrumbItem, Divider, PageSection, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { RBACStore } from '../../redux/store';
+import { useParams } from 'react-router-dom';
+import AppTabs from '../app-tabs/app-tabs';
+import pathnames from '../../utilities/pathnames';
+import WorkspaceActions from './WorkspaceActions';
+import Messages from '../../Messages';
+
+interface WorkspaceData {
+  name: string;
+  uuid: string;
```

Comment:
the uuid has been changed to id recently.
```suggestion
  id: string;
```

Reply:
Fixed